extension/src/bs/MessageForwardHandler.ts

```diff
@@ -0,0 +1,75 @@
+import * as net from "net";
+import * as rpc from "vscode-jsonrpc/node";
+import * as vscode from "vscode";
+import * as path from "path";
+import { generateRandomPipeName } from "../util/generateRandomPipeName";
+
+export const GET_IMPORTER_PIPE_NAME = "gradle.getImporterPipeName";
+
+export class MessageForwardHandler {
+    private buildServerConnection: rpc.MessageConnection | null = null;
```

Comment:
Name it as MessageProxy, containing two class instance `BuildServerConnection` & `JdtlsImporterConnection`